charts/barman/templates/configmap-barman-backups.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "barman.fullname" . }}-backups-config
+  labels:
+    {{- include "barman.labels" . | nindent 4 }}
+data:
+  {{- range .Values.barman.backups }}
+  {{ .postgresql.host }}.conf: |
```

Comment:
What happens if you configure multiple barman configs with the same hostname?

Reply:
Good question. What would be the use case here? AFAIK Barman only supports backups of full PostgreSQL instances.

Reply:
One possible use case I could image is having separate backup configs with different retention times, but I don't think that's a feature we would really need.